.github/workflows/release.yaml

```diff
@@ -0,0 +1,46 @@
+name: Release
+on:
+  push:
+    tags:
+      - "*"
```

Comment:
Question for Paul - how do you want to create a release?  You can make a tag and push it to git which will kick off this release workflow and release a semtk-opensource-{{tag}}-bin.tar.gz (everything else will be thrown away by the way - I ask you further below if you'd rather deploy everything to Sonatype).  Alternatively, you can go to the Releases page, click the `Draft a new release` button, choose whichever version/tag you want, fill out the rest of the form, and then submitting the form will kick off this workflow if we change its beginning from `on / push / tags` to 

```
on:
  release:
    types: [ published ]
```